pkg/template/generator/expressionvalue.go

```diff
@@ -94,6 +95,8 @@ func replaceWithGenerated(s *string, expression string, ranges [][]byte, length
 			alphabet += Numerals
 		case `\a`:
 			alphabet += Alphabet + Numerals
+		case `\S`:
```

Comment:
@liggitt i'm open for suggestions... I thought \A might be better, in regexp that will mean negation of \a which we use for aphabet + numbers